bin/extract-ORFs.py

```diff
@@ -39,20 +39,35 @@
 
     parser.add_argument(
         '--minORFLength', metavar='LEN', type=int, default=None,
-        help='Only ORFs of at least this length will be written to stdout.')
+        help=('Only ORFs of at least this length will be written to stdout.'))
 
     parser.add_argument(
         '--maxORFLength', metavar='LEN', type=int, default=None,
-        help='Only ORFs of a maximum of this length will be written to'
-        'stdout.')
+        help=('Only ORFs of a maximum of this length will be written to '
+              'stdout.'))
+
+    group = parser.add_mutually_exclusive_group()
+
+    group.add_argument(
+        '--kozakOnly', default=False, action='store_true',
+        help=('Only ORFs that also have a Kozak consensus will be written to '
+              'stdout. A file with Kozak consensus information will be '
```

Comment:
I don't think this "A file with..." part is true, is it?  That file is only written in `--kozakInfoFile` is used?

Reply:
yes, changed

Reply:
Also changed: kozakInfoFile and KozakOnly can be used together